AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -169,13 +218,38 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
 
     changeDisplayName() {
         if (this.judge && this.judge.display_name) {
-            const indexOfJudge = this.hearing.participants.findIndex(x => x.is_judge === true);
+            const indexOfJudge = this.hearing.participants.findIndex(x => x.is_judge);
             if (indexOfJudge !== -1) {
                 this.hearing.participants[indexOfJudge].display_name = this.judge.display_name;
             }
         }
-        const text = SanitizeInputText(this.judgeDisplayName.value);
-        this.judgeDisplayName.setValue(text);
+        const text = SanitizeInputText(this.judgeDisplayNameFld.value);
+        this.judgeDisplayNameFld.setValue(text);
+    }
+
+    changeEmail() {
+        console.log('change email', this.otherInformationDetails.judgeEmail);
+        let judgeEmail = this.otherInformationDetails.judgeEmail;
+        if (this.otherInformationDetails.judgeEmail === '') {
+            judgeEmail = this.courtAccountJudgeEmail;
+        }
+        const indexOfJudge = this.hearing.participants.findIndex(x => x.is_judge);
```

Comment:
recommend using a `find` here, returns an obj which you can then just check is truthy